.github/workflows/build-and-test.yml

```diff
@@ -5,11 +5,14 @@ jobs:
   build-and-test:
     name: "Test on Python ${{ matrix.python-version}}"
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
Why do we want to not cancel the whole matrix if a single job fails?

Reply:
I think the point is continuing after a failure on 3.10 or 3.11. It *must* work on 3.7 and whatever version bullseye has. 3.9?

Reply:
But would we realistically merge a PR that fails in at least one of the python versions? Or is it more so that we can get an overview if it fails on all versions or just one?

Reply:
I've removed the commit that adds this.

Reply:
@johannaengland As you say, it's not about merging, it's about getting an overview of everything that fails, instead of cancelling the remaining test suites when a single one fails.

But the change doesn't work as intended.  As demonstrated by this very PR, if *all* the test suites failed, the matrix was still considered successful and the check became green.  Which is why we agreed to just remove this change for now.